src/Cassandra.IntegrationTests/Core/TimeUuidSerializationTests.cs

```diff
@@ -71,6 +74,24 @@ public void DeserializationTests()
         }
 
         [Test]
+        public void TimeUuid_Should_Execute_TimeUuid_CQL_Functions()
+        {
+            var guid = Guid.NewGuid();
+            var timeuuidSample = TimeUuid.NewId();
+            var dateOffset = timeuuidSample.GetDate();
+
+            var selectMinMaxTimeuuidPrepared = Session.Prepare(string.Format("select * from {0} where id = ? " +
+                                                                         "and timeuuid_sample < maxTimeUuid(?) and timeuuid_sample > minTimeuuid(?)",
```

Comment:
Instead of using CQL functions `maxTimeUuid()` and `minTimeUuid()` for the tests, we should be using client-side C# functions `TimeUuid.Max()` and `TimeUuid.Min()`.